ext/ctype/ctype.stubs.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+function ctype_alnum(string $text): bool {}
```

Comment:
`$text` can actually be [any type](https://github.com/php/php-src/blob/9436a2c488d593aca8d1912433ca3b14f99b1c39/ext/ctype/ctype.c#L145-L175), so
```suggestion
function ctype_alnum($text): bool {}
```
Same for the other ctype functions.

Reply:
Done!